pages_project_test.go

```diff
@@ -583,9 +613,12 @@ func TestUpdatePagesProject(t *testing.T) {
 
 	mux.HandleFunc("/accounts/"+testAccountID+"/pages/projects/Test Pages Project", handler)
 
-	_, err := client.UpdatePagesProject(context.Background(), testAccountID, "Test Pages Project", *updateAttributes)
+	_, err := client.UpdatePagesProject(context.Background(), AccountIdentifier(""), *updateAttributes)
```

Comment:
this test will be broken at the moment while i chat with @WalshyDev about what we are doing with friendly URL IDs since these shouldn't be able to change but i think we currently allow them to.